tools/swift-format/Sources/Rules/NoCasesWithOnlyFallthrough.swift

```diff
@@ -11,5 +11,97 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#fallthrough-in-switch-statements
 public final class NoCasesWithOnlyFallthrough: SyntaxFormatRule {
+  
+  public override func visit(_ node: SwitchStmtSyntax) -> StmtSyntax {
+    var newCases: [SwitchCaseSyntax] = []
+    var violations: [SwitchCaseLabelSyntax] = []
+    
+    for switchCase in node.cases {
+      guard let switchCase = switchCase as? SwitchCaseSyntax else { continue }
+      guard let label = switchCase.label as? SwitchCaseLabelSyntax else { newCases.append(switchCase); continue }
+      
+      if switchCase.statements.count == 1,
+        let only = switchCase.statements.first,
+        only.item is FallthroughStmtSyntax {
+        violations.append(label)
+      } else {
+        guard violations.count > 0 else { newCases.append(switchCase); continue }
+        if retrieveNumericCaseValue(caseLabel: label) > 0 {
+          let newCase = collapseIntegerCases(violations: violations, validCaseLabel: label, validCase: switchCase)
+          newCases.append(newCase)
+        } else {
+          let newCase = collapseNonIntegerCases(violations: violations, validCaseLabel: label, validCase: switchCase)
+          newCases.append(newCase)
+        }
+        violations = []
+      }
+    }
+    return node.withCases(SyntaxFactory.makeSwitchCaseList(newCases))
+  }
+  
+  // Puts all given cases on one line with range operator or commas
+  func collapseIntegerCases(violations: [SwitchCaseLabelSyntax], validCaseLabel: SwitchCaseLabelSyntax, validCase: SwitchCaseSyntax) -> SwitchCaseSyntax {
+    var isConsecutive = true
+    var index = 0
+    var caseNums: [Int] = []
+    
+    for item in violations { caseNums.append(retrieveNumericCaseValue(caseLabel: item)) }
+    caseNums.append(retrieveNumericCaseValue(caseLabel: validCaseLabel))
+    
+    while index <= caseNums.count - 2, isConsecutive {
+      isConsecutive = caseNums[index] + 1 == caseNums[index + 1]
+      index += 1
+    }
 
+    var newCaseItems: [CaseItemSyntax] = []
+    let first = caseNums[0]
+    let last = caseNums[caseNums.count - 1]
+    if isConsecutive {
+      // Create a case with a sequence expression based on the new range
+      let start = SyntaxFactory.makeIntegerLiteralExpr(digits: SyntaxFactory.makeIntegerLiteral("\(first)"))
```

Comment:
Can you make sure to format these lines according to the style guide? Specifically the 100-character line limit and wrapping function call arguments.